### DISCUSSION TITLE
Amazfit bip u pro

### DISCUSSION THREAD

**Discussion by bbzen** [NightscoutFoundation/xDrip#3231] [Ideas]
Hello, could you add sending bg from xdrip to Amazfit bip u pro. Fine watch but xdrip doesn't support them. Thanks in advance.

**Comment by PhilTU** [comment]
Support for Amazfit devices is using the Watchdrip+ application developed by Artem 

https://bigdigital.home.blog/2022/12/21/zepp-os-devices-support/ - ZeppOS devices
https://bigdigital.home.blog/2022/06/16/watchdrip-a-new-application-for-xdrip-watch-integration/ - Mifit / Xiomi devices

there are some bip devices listed as compatible in the second link

**Reply by GBRacerX1** [reply]
do they still require a phone

**Reply by PhilTU** [reply]
Yes. Sensor -> xDrip+ on phone -> Watchdrip+ on phone -> Watch
